wandb/sdk/wandb_init.py

```diff
@@ -467,6 +468,7 @@ def init(
     force: Optional[bool] = None,
     tensorboard=None,  # alias for sync_tensorboard
     sync_tensorboard=None,
+    logdir=None,
```

Comment:
If we add this we should likely make this `tensorboard_logdir` as we already have a `dir` arg.  We currently have 3 options:

1. Don't add anything to `wandb.init` and make the error message tell the user to call `wandb.tensorboard.patch(logdir="...")` before calling wandb.init
2. Make this tensorboard_logdir
3. Make sync_tensorboard optionally be a string

I'll leave it to @raubitsj to decide.

Reply:
Okay! Sounds good, will wait for Jeff to chime in, then fix it.

Reply:
(1) or (3)
I think we should do (1) for now and consider what other tensorboard controls we want before choosing something like (3)

Reply:
Okay, did 1 for now, we can come back around for 3